conf/application.conf

```diff
@@ -42,6 +42,10 @@ db.default.url="jdbc:postgresql://localhost:5432/bacardi"
 db.default.user=bacardi
 db.default.password=batman
 
+db.default.url=${?DB_URL}
+db.default.user=${?DB_USER}
```

Comment:
The documentation says they leave them alone if they don't exist. I don't override `DB_URL`, so that seems to be the case.